sig-etcd/README.md

```diff
@@ -46,6 +46,10 @@ subprojects, and resolve cross-subproject technical issues and decisions.
 ## Subprojects
 
 The following [subprojects][subproject-definition] are owned by sig-etcd:
+### auger
+Directly access data objects stored in etcd by kubernetes.
+- **Owners:**
+  - [etcd-io/auger](https://github.com/etcd-io/auger/blob/master/OWNERS)
```

Comment:
can we rename `master` to `main`, to be consistent with all projects under etcd-io?
```suggestion
  - [etcd-io/auger](https://github.com/etcd-io/auger/blob/main/OWNERS)
```